src/pages/Register.js

```diff
@@ -35,7 +35,7 @@ const Register = () => {
         lastName: familyName,
         contactEmail: email
       })
-      authStore.update(state => ({ ...state, registrationStatus: 'registered' }))
+      authStore.update(state => ({ ...state, registrationStatus: userStatus.createdProfile }))
```

Comment:
This code was the cause of the Ajax calls where `registrationStatus` was `registered` and yet `acceptedTos` was `false`. We should not be assuming `registered` here as the full workflow is not yet complete.